cloud_blobstore/__init__.py

```diff
@@ -1,6 +1,74 @@
 import typing
 
 
+class PagedIter:
+    """
+    Provide an iterator that will iterate over every object, filtered by
+    prefix and delimiter. Alternately continue iteration with token and
+    key (marker).
+    """
+
+    def __init__(
+            self,
+            prefix: str=None,
+            delimiter: str=None,
+            marker: str=None,
```

Comment:
I feel like the behavior of marker is really hard to understand from the variable name or documentation.

Reply:
I was following AWS terminology, but I agree.